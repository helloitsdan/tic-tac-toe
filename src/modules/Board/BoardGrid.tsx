import React, { FunctionComponent } from 'react'
import { ReactComponent as BoardIcon } from '../../assets/board.svg'

const BoardGrid: FunctionComponent = () => (
  <BoardIcon className="c-game-board__grid" />
)

export default BoardGrid
