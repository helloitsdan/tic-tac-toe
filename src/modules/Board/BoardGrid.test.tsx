import React from 'react'
import { render } from '@testing-library/react'

import BoardGrid from './BoardGrid'

describe('BoardGrid', () => {
  it('renders', () => {
    const { asFragment } = render(<BoardGrid />)
    expect(asFragment()).toMatchSnapshot()
  })
})
