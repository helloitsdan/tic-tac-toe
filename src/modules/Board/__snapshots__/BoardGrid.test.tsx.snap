// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardGrid renders 1`] = `
<DocumentFragment>
  <svg
    class="c-game-board__grid"
  >
    board.svg
  </svg>
</DocumentFragment>
`;
